impress-org/givewp
# Feature: Show "Transfer" menu item to user when transfer bar is dismissed

## Description
After a user migrates a v2 form to a v3 form they're shown the builder. At the bottom of the builder is a blue bar telling them how to transfer data. If they dismiss that bar, a "Transfer Donation Data"  menu item shows up in the site options.

This PR is adding functionality that will open the sidebar menu and show the user the menu item so they know how to transfer when they're ready.


## Visuals

![Screen Capture_select-area_20230919202339](https://github.com/impress-org/givewp/assets/4222590/fea902ce-8403-4ab9-9aaf-5385b3312a11)


## Testing Instructions

- Add form
- Upgrade the form
- Dismiss the transfer bottom bar

When the transfer bottom bar is dismissed, the sidebar menu should be opened and the Transfer Donation Data menu item should be selected. Also, the tooltip which explains what that menu item does should be visible (see attached video)



## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205502412661044
  - https://app.asana.com/0/0/1205550233964861
